### ISSUE TITLE
[i18n-TR] Translating docs to Turkish

### ISSUE THREAD

**Issue by mertyyanik** [huggingface/transformers#27088]
Hi!

Let's bring the documentation to all the Turkish-speaking community 🌐 (currently 0 out of 267 complete)

Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know in this issue if you'd like to translate any, and we'll add your name to the list.

Some notes:

* Please translate using an informal tone (imagine you are talking with a friend about transformers 🤗).
* Please translate in a gender-neutral way.
* Add your translations to the folder called `tr` inside the [source folder](https://github.com/huggingface/transformers/tree/main/docs/source).
* Register your translation in `tr/_toctree.yml`; please follow the order of the [English version](https://github.com/huggingface/transformers/blob/main/docs/source/en/_toctree.yml).
* Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @stevhliu and @MKhalusova for review.
* 🙋 If you'd like others to help you with the translation, you can also post in the 🤗 [forums](https://discuss.huggingface.co/).

## Get Started section

- [ ] [index.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/index.md) https://github.com/huggingface/transformers/pull/20180
- [ ] [quicktour.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/quicktour.md) (waiting for initial PR to go through)
- [ ] [installation.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/installation.md).

## Tutorial section
- [ ] [pipeline_tutorial.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/pipeline_tutorial.md)
- [ ]  [autoclass_tutorial.md](https://github.com/huggingface/transformers/blob/master/docs/source/autoclass_tutorial.md)
- [ ]  [preprocessing.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/preprocessing.md)
- [ ]  [training.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/training.md)
- [ ]  [accelerate.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/accelerate.md)
- [ ]  [model_sharing.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_sharing.md)
- [ ]  [multilingual.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/multilingual.md)

**Comment by mertyyanik** [comment]
I will work index.md first.

**Comment by Dilssssss** [comment]
hi there i will be working on translating pipeline_tutorial.md

**Comment by Dilssssss** [comment]
hey @mertyyanik is it okay if i start translating pipeline_tutorial.md ?

**Comment by Dilssssss** [comment]
hey @mertyyanik  what are the next steps  😁

**Comment by Crontex169** [comment]
Hello, what happened to translation?. It looks like it is removed. @mertyyanik

**Comment by onwp** [comment]
> Hello, what happened to translation?. It looks like it is removed. [@mertyyanik](https://github.com/mertyyanik)

Correct. It was removed. I sent a new PR for that. So it should be coming soon enough.

**Comment by nikirun2017-phache** [comment]
For what it's worth, I've been using [Summon Translator](https://summontranslator.com) to handle translation requests. You point it at a PR URL and it translates the i18n files while keeping the markdown formatting intact — headers, code blocks, bullet lists all survive the translation.

The main thing that makes it usable vs. copy-pasting into Google Translate is that it handles technical terminology correctly (using Claude or GPT-4o under the hood) and doesn't strip the markdown structure. A PR with a carefully formatted description doesn't come back as a wall of unformatted text.

It's pay-per-use so there's no ongoing cost if you only need it occasionally. First 1,000 words are free.

Might save you the back-and-forth of asking contributors to resubmit in English.
